docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -83,7 +83,7 @@ With Grafana 10.1 browsing log lines is faster than ever before after a series o
 
 _Generally available in all editions of Grafana._
 
-Load as many log lines as you need in logs context just by scrolling through them.
+Log context allows you to view additional lines of of context surrounding a given log line. With this change you can see as many log lines as you need in log context: as you scroll through the log lines, we load in more log lines, as needed.
```

Comment:
I like your suggestion @ivanahuckova! I've added a couple tweaks. We want to avoid enable/disable language where we can. Wondering if we can avoid a bit of repetition in that last sentence.

```suggestion
Log context allows you to view additional lines surrounding a specific log entry. With this enhancement, you can access as many log lines as needed within the log context. As you scroll through the logs, Grafana dynamically loads more log lines, ensuring a seamless and continuous viewing experience.
```

Reply:
thanks, applied 👍